Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/EventTag/ByEventTag.tsx

```diff
@@ -81,6 +82,12 @@ const EventTags: Application.Types.iByComponent = (props) => {
         setErrors(e);
     }, [record]);
 
+    const deleteTag = React.useCallback(() => {
+        dispatch(EventTagSlice.DBAction({ verb: 'DELETE', record }));
+        setShowWarning(false);
+        setMode('View');
```

Comment:
These two set states are unnecessary. These are already set in the callback for the warning and modal, respectively.